ydb/mvp/oidc_proxy/openid_connect.h

```diff
@@ -10,16 +11,39 @@
 #include <library/cpp/string_utils/base64/base64.h>
 
 struct TOpenIdConnectSettings {
-    static const inline TString CLIENT_ID = "yc.oauth.ydb-viewer";
     static const inline TString YDB_OIDC_COOKIE = "ydb_oidc_cookie";
+    static const inline TString SESSION_COOKIE = "session_cookie";
+
+    static const inline TString AUTH_REQUEST_Y = "/oauth/authorize";
+    static const inline TString AUTH_REQUEST_N = "/oauth2/authorize";
+    static const inline TString TOKEN_REQUEST_Y = "/oauth/token";
+    static const inline TString TOKEN_REQUEST_N = "/oauth2/token";
+    static const inline TString EXCHANGE_REQUEST = "/oauth2/session/exchange";
+
+    TString ClientId;
     TString SessionServiceEndpoint;
     TString SessionServiceTokenName;
     TString AuthorizationServerAddress;
     TString ClientSecret;
     std::vector<TString> AllowedProxyHosts;
+    NMVP::EAuthProfile AuthProfile = NMVP::EAuthProfile::YProfile;
 
     TString GetAuthorizationString() const {
-        return Base64Encode(CLIENT_ID + ":" + ClientSecret);
+        return Base64Encode(ClientId + ":" + ClientSecret);
+    }
+
+    TString GetAuthEndpoint() const {
```

Comment:
Can we move these values into config with default values that were harcoded here?
For new settings they will be able to be set through config

Reply:
yes, we can